cmd/kube-apiserver/app/server.go

```diff
@@ -118,6 +124,8 @@ cluster's shared state through which all other components interact.`,
 	fs := cmd.Flags()
 	namedFlagSets := s.Flags()
 	verflag.AddFlags(namedFlagSets.FlagSet("global"))
+	utilversion.DefaultComponentGlobalsRegistry.AddFlags(namedFlagSets.FlagSet("global"))
```

Comment:
I expected to see this done under `s.Flags`, when delegated to `s.Options.AddFlags`